media/stylus/components/share-link.styl

```diff
@@ -0,0 +1,122 @@
+/*
+share-link
+====================================================================== */
+.share-link {
+    border: 1px solid rgba-fallback(rgba(255,255,255,0.2));
+    border-radius: 2px;
+    box-shadow: none;
+    color: #fff;
+    bidi-value(margin, 0px 6px 6px 0px, 0px 0px 6px 6px);
```

Comment:
I'm pretty sure our compressor does this, but just to be safe, maybe we could replace `0px` with `0` in this file.